zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -14,6 +15,21 @@ object HttpData {
   final case class CompleteData(data: Chunk[Byte])             extends HttpData[Any, Nothing]
   final case class StreamData[R, E](data: ZStream[R, E, Byte]) extends HttpData[R, E]
 
+  sealed trait FormDataContent extends Product with Serializable
+
+  final case class FileData(name: String, contentType: String, content: ZStream[Blocking, Throwable, Byte])
+      extends FormDataContent
+  final case class AttributeData(name: String, content: ZStream[Blocking, Throwable, Byte]) extends FormDataContent
+
+  object MultipartFormData {
+    def empty: MultipartFormData = MultipartFormData(Map.empty, Map.empty)
+  }
+
+  final case class MultipartFormData(
+    attributes: Map[String, AttributeData],
+    files: Map[String, FileData],
+  ) extends HttpData[Any, Nothing]
```

Comment:
I think we should add `Blocking` and `Throwable` to HttpData.

```suggestion
  ) extends HttpData[Blocking, Throwable]
```